x/subscription/keeper/subscription.go

```diff
@@ -209,6 +209,21 @@ func (k Keeper) advanceMonth(ctx sdk.Context, subkey []byte) {
 		return
 	}
 
+	blocksToSave, err := k.epochstorageKeeper.BlocksToSave(ctx, block)
+	if err != nil {
+		utils.LavaFormatError("critical: failed assigning CU tracker callback, skipping", err,
+			utils.Attribute{Key: "block", Value: block},
+		)
+	} else {
+		cuTrackerTimerKey := sub.Consumer + " "
+		if sub.DurationLeft <= 1 {
+			cuTrackerTimerKey += "0" // mark that the subscription is expired, so trackedCu object needs to be removed
```

Comment:
why dont we use the data bytes instead of adding it to the key?

Reply:
done. see https://github.com/lavanet/lava/commit/93dc7f6261cac796b09559e5fa4c33be54361ac4